src/utils/sanityUtils.ts

```diff
@@ -2,16 +2,22 @@ import {
 	SanityTekstObjekt,
 	SanityChildren,
 	SanityDropdown,
+	SanityTekst,
 } from '../typer/sanity';
 
-export const sanityToHtml = (tekst: SanityTekstObjekt): string[] => {
+export const sanityBlocktekstToHtml = (tekst: SanityTekstObjekt): string[] => {
 	const outStrengTabell: string[] = [];
 	let gjeldendeTabellElement = '';
 	tekst.tekst.forEach((tekstElement) => {
 		tekstElement.children.forEach((child, childIndeks) => {
+			const childErFlettefeltReferanse = erFlettefeltReferanse(
```

Comment:
Litt pirkete, men vil erChildFlettefeltReferanse være et bedre navn? Lettere å skjønne at noe er boolsk når det begynner med is/er